### PR TITLE
new deprecated objects page and menu item

### PR DESCRIPTION
**Pull request by lenapivotal**
Created new page as requested by Nirali and gpupgrade team. 
Plan is to decommission https://gpdb.docs.pivotal.io/upgrade/1-0/guc-changes-5to6.html when all info is migrated in the gpdb docs upgrade/migrate sections.

Tracker: https://www.pivotaltracker.com/story/show/178371453

temporary HTML version (needs VPN) :
https://docs-lena-deprecated-objects.sc2-04-pcf1-apps.oc.vmware.com/7-0/ref_guide/deprecated-objects.html
https://docs-lena-deprecated-objects.sc2-04-pcf1-apps.oc.vmware.com/7-0/install_guide/migrate.html

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/ref_guide.ditamap [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
 <map>
-	<topicref href="../../6-15/homenav.html" scope="external"
-		navtitle="Greenplum Database 6.15 Documentation" format="html" otherprops="oss-only"/>
-	<topicref href="../../6-15/homenav.html" scope="external"
-		navtitle="Tanzu Greenplum 6.15 Documentation" format="html" otherprops="pivotal"/>
+	<topicref href="../../7-0/homenav.html" scope="external"
+		navtitle="Greenplum Database® 7.0 Beta Documentation" format="html" otherprops="oss-only"/>
+	<topicref href="../../7-0/homenav.html" scope="external"
+		navtitle="Tanzu Greenplum® 7.0 Beta Documentation" format="html" otherprops="pivotal"/>
```
Comment (soumyadeep2007) [inline gpdb-doc/dita/ref_guide/ref_guide.ditamap:7]: The 7.0s here confused me since this is for 6?

**Review by soumyadeep2007 — APPROVED** [review]

File: gpdb-doc/dita/ref_guide/ref_guide.ditamap (hunk 1 above)
Comment (lenapivotal) [reply gpdb-doc/dita/ref_guide/ref_guide.ditamap:7]: Sorry, ignore the menu/versioning. I just build the app in the latest book, but the page will only be merged with 6X_STABLE.

**Review by niralis13 — APPROVED** [review]

File: gpdb-doc/dita/install_guide/migrate.xml [hunk 2]
```diff
@@ -2,15 +2,16 @@
 <!DOCTYPE topic
   PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
 <topic id="topic1" xml:lang="en">
-  <title id="kh138244">Migrating Data from Greenplum 4.3 or 5 to Greenplum 6</title>
+  <title id="kh138244">Migrating or Upgrading from Greenplum 4.3 or 5 to Greenplum 6</title>
```
Comment (dyozie) [inline gpdb-doc/dita/install_guide/migrate.xml:5]: We might consider shortening the title to just "Migrating or Upgrading to Greenplum 6" and then describing the gory details in the text.

File: gpdb-doc/dita/install_guide/migrate.xml [hunk 3]
```diff
@@ -97,8 +98,11 @@
     </body>
   </topic>
   <topic id="prep-gpdb4">
-    <title>Preparing Greenplum 4.3 and 5 Databases for Backup</title>
+    <title>Preparing Greenplum 4.3 and 5 Databases for Backup and Upgrade</title>
     <body>
+      <p>This section highlights changes in Greenplum 6.x that you need to be aware when upgrading
```
Comment (dyozie) [inline gpdb-doc/dita/install_guide/migrate.xml:103]: aware -> aware of

Comment (dyozie) [reply gpdb-doc/dita/install_guide/migrate.xml:103]: and plan to restore -> with the intention of restoring

**Review by dyozie — APPROVED** [review]

**Comment by lenapivotal** [comment]
I have replicated the content to https://github.com/greenplum-db/gpdb/pull/12331 as this branch was based from master, which is gpdb 7 content so messy to merge.

**Pull request closed without merging**